components/Chips/tests/unit/ChipViewFontThemerTests.m

```diff
@@ -29,7 +29,7 @@ @implementation ChipViewFontThemerTests
 - (void)testFontThemer {
   // Given
   MDCChipView *chip = [[MDCChipView alloc] init];
-  MDCBasicFontScheme *fontScheme = [[MDCBasicFontScheme alloc] init];
+  MDCTypographyScheme *fontScheme = [[MDCTypographyScheme alloc] init];
```

Comment:
In a follow-up we'll want to do a global find-and-replace for "fontScheme"

Reply:
FIrst pass.